src/main/webapp/dataUse/DataUseRestriction.js

```diff
@@ -0,0 +1,181 @@
+import React, { Component } from 'react';
+import { h, h1, div, a, hh } from 'react-hyperscript-helpers';
+import { DataUse } from "../util/ajax";
+import { spinnerService } from "../util/spinner-service";
+import { Spinner } from "../components/Spinner";
+import { TableComponent } from "../components/TableComponent";
+import { RESTRICTION_SORT_NAME_INDEX, styles } from "../util/ReportConstants";
+import { TABLE_ACTIONS } from "../util/TableUtil";
+import { isEmpty } from "../util/Utils";
+import { Link } from 'react-router-dom';
+import { format } from 'date-fns';
+
+const SIZE_PER_PAGE_LIST = [
+  { text: '50', value: 50 },
+  { text: '100', value: 100 },
+  { text: '500', value: 500 }];
+
+const columns = [
+  {
+    dataField: 'consentGroupKey',
+    text: 'Consent Group',
+    sort: true,
+    formatter: (cell, row, rowIndex, colIndex) =>
+    div({},[
+      h(Link, {to: {pathname:'/newConsentGroup/main', search: '?consentKey=' + row.consentGroupKey, state: {issueType: 'consent-group', tab: 'documents', consentKey: row.consentGroupKey}}}, [row.consentGroupKey])
+    ])
+  },
+  {
+    dataField: 'id',
+    text: 'View Restrictions',
+    sort: false,
+    formatter: (cell, row, rowIndex, colIndex) =>
+    div({},[
+      a({href: component.serverURL + '/dataUse/show/' + row.id}, ['View Restriction'])
```

Comment:
Shouldn't this also be a `Link`?

Reply:
I can't use Link  because Data Use Restriction view page has not been migrated yet. I'll do that on US 748

Reply:
👍